essentials.sh

```diff
@@ -107,6 +107,30 @@ def_var() {
     >&2 echo "+ ${var_name}=${value}"
 }
 
+# Acts like def_var but additionally requires that the target value is Boolean.
+# The function is a kind of shortcut for
```

Comment:
Remove these lines. They belong to `def_var`, there is no need to repeat it again.